README.md

```diff
@@ -41,7 +41,7 @@ Also, a serializer needs to be bound to this trait in a configuration file:
 ```scala
 akka.actor {
   serializers {
-    jackson-json = "akka.serialization.jackson.JacksonJsonSerializer"
+    jackson-json = "org.apache.pekko.serialization.jackson.JacksonJsonSerializer"
```

Comment:
TBD if such class exists

Reply:
Yes it does: https://pekko.apache.org/docs/pekko/current/serialization.html#configuration